unit/src/EBSDMeshErrorTest.C

```diff
@@ -28,7 +28,7 @@ void
 EBSDMeshErrorTest::setUp()
 {
   const char *argv[2] = { "foo", "\0" };
-  _app = AppFactory::createApp("MarmotUnitApp", 1, (char**)argv);
+  _app = AppFactory::createApp("MooseUnitApp", 1, (char**)argv);
```

Comment:
If we change the unit tests, should we actually run them before committing the changes?